WordPress/src/main/java/org/wordpress/android/models/JetpackSettingsModel.java

```diff
@@ -1,17 +1,27 @@
 package org.wordpress.android.models;
 
+import java.util.ArrayList;
+
 public class JetpackSettingsModel {
     public long localTableId;
     public boolean monitorActive;
     public boolean emailNotifications;
     public boolean wpNotifications;
+    public boolean jetpackProtectEnabled;
+    public final ArrayList<String> jetpackProtectWhitelist = new ArrayList<>();
```

Comment:
I did it to avoid including both `List` and `ArrayList` dependencies, but I agree that it's weird.